mod/deckorate_datasets/set/type/dataset.rb

```diff
@@ -9,14 +10,38 @@
 card_reader :data_subset, type: :search_type
 card_reader :wikirate_topic, type: :list
 
-def parent_dataset_card
-  Card[parent_dataset]
+format :html do
+  def breadcrumb_items
+    breadcrumb_array = [
+      link_to("Home", href: "/"),
+      link_to_card(:dataset, "Datasets"),
+      render_name
+    ]
+
+    insert_parent_link(breadcrumb_array) if should_insert_parent_link?
+
+    breadcrumb_array
+  end
+
+  private
+
+  def should_insert_parent_link?
+    card.parent != "" && card
```

Comment:
I don't think card will ever be nil.

You could just do `card.parent.present?`